### PR TITLE
Tokemak abstractions - track more data (pt.2)

### PR DESCRIPTION
**Pull request by hivtushok**
_**[This is a part 2 of PR #1973]**_

Brief comments on the purpose of your changes:

This PR updates Tokemak queries to help reflect changes to the dashboards. Includes:

- Tracking of Convex Frax FRAX/USDC & FXS/crvFXS metapools

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Comment by hivtushok** [comment]
@antonio-mendes @agaperste @jeff-dude @dot2dotseurat please, review this part two PR.

File: deprecated-dune-v1-abstractions/ethereum/tokemak/view_tokemak_wallet_balances_daily.sql [hunk 1]
```diff
@@ -110,6 +110,35 @@ WITH calendar AS
         --GROUP BY source, contract_address,symbol,display_name
         --order by "date" desc, symbol
         UNION
+        --CONVEX FRAX
+        SELECT "date", source,wallet_address, token_address, symbol,display_name, sum(qty) OVER (PARTITION BY wallet_address,symbol ORDER BY "date")as balance 
+        FROM (
+                SELECT "date", 5 as source,wallet_address, contract_address as token_address, symbol,display_name, sum(qty) as qty FROM (
```
Comment (jeff-dude) [inline deprecated-dune-v1-abstractions/ethereum/tokemak/view_tokemak_wallet_balances_daily.sql:116]: this subquery is failing. please do run it on dune & fix the errors

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: deprecated-dune-v1-abstractions/ethereum/tokemak/view_tokemak_wallet_balances_daily.sql (hunk 1 above)
Comment (hivtushok) [reply deprecated-dune-v1-abstractions/ethereum/tokemak/view_tokemak_wallet_balances_daily.sql]: Should be good now, thanks

**Comment by jeff-dude** [comment]
i'm unable to push changes here, so i opened #2217 

i am using an orchestrator script to run these, since they are deeply nested and dropping leads to rebuild of many others too. for future fixes, please either edit this file (it's just copy/paste of each script to orchestrator file in correct order of operations) or grant me access to your forked repo to push changes

**Pull request closed without merging**